test/man/checkManPages

```diff
@@ -137,6 +137,19 @@ def check_man_page(bin_name, additional_args=None):
         helpflags = helpsections[section]
         manflags = mansections[section]
 
+        # Check if the manpage flags of this section have '\' , otherwise throws error
+        for mflag in manflags:
+            if mflag[0] == "\\" or mflag[4] == "\\":
+                continue
+            else:
+                numerrors+=1
+                _error('man section "{0}" has some misformatted flags:\
+                        {1}'.format(section, mflag))
+
+        # Removing '\' from manpage flags of this section to match with compiler's flags      
+        for i in range(len(manflags)):
+            manflags[i] = manflags[i].replace("\\", "")   
```

Comment:
Hi @shubhamkmr04 — First, thanks for jumping on this!  Two questions with the above:

* what is the relevance of the `mflag[4]` case?  When does that come up?  (and assuming it's important, maybe note why in the comment?
* similarly, I wondered whether in lines 150-151, there'd be any reason to only replace the mflag[0] backslashes (and/or mflag[4] if I understood it).  I don't know that there are cases where flags would want to use `\` elsewhere (and would hope not), so this is probably fine, but it seemed like it could be preferable to be similarly precise or not in both places?  (to that end, could the `.replace()` call somehow occur within the `continue` clause above so that the logic was more like "if we find the backslash, that's good, and let's remove it before going on?"  My Python isn't good enough to know how easy/hard this would be.